src/gui/widgets/text_box_base.cpp

```diff
@@ -478,11 +478,27 @@ void text_box_base::signal_handler_sdl_key_down(const event::ui_event event,
 
 // For copy/paste we use a different key on the MAC. Other ctrl modifiers won't
 // be modifed seems not to be required when I read the comment in
-// widgets/textbox.cpp:516. Would be nice if somebody on a MAC would test it.
+// widgets/textbox.cpp:495. But on macOS I encourage turning the modifier key
+// to be 'cmd' (instead of 'ctrl') for 'a' (select all) as well, to have all
+// functionality in text boxes as most aligned in a same direction as possible,
+// which in the case of macOS it likely should be using vendor defaults, for
+// now.
+
+// The election for modifier key is not pervasive and, even though configurable
+// (or at least it should be configurable), is sometimes set to what the
+// desktop environment defines as idiomatic, but one should bear in mind it
+// could be largely remapped by the user.
+
 #ifdef __APPLE__
-	const unsigned copypaste_modifier = KMOD_LGUI | KMOD_RGUI;
+	// Idiomatic modifier keys in macOS computers for some _gestures_.
+	const SDL_Keycode copypaste_modifier = KMOD_GUI;
+	const SDL_Keycode select_all_modifier = KMOD_GUI;
 #else
-	const unsigned copypaste_modifier = KMOD_CTRL;
+	// Idiomatic modifier keys in Microsoft desktop environments for some
+	// _gestures_. _Common_ in GNU/Linux desktop environments as well.
```

Comment:
Why the extra underscores in here? I guess it doesn't matter _that_ much, but it makes it look like you're treating the comment content as Markdown, which doesn't make any sense.

Reply:
Wouldn't have made any sense unless Markdown had been conceived specifically to provide source readability... but that's the way they designed it.